src/files-and-videos/generic/ApiStatusToast.jsx

```diff
@@ -21,7 +21,7 @@ const ApiStatusToast = ({
 
   return (
     <Toast
-      show={isOpen}
+      show={(actionType === 'add' && fileType === 'video') ? false : isOpen}
```

Comment:
I would make this change in `FilesTable` [LN 247-254](https://github.com/openedx/frontend-app-course-authoring/blob/master/src/files-and-videos/generic/FileTable.jsx#L247-L254). This file is a generic file so the any render conditionals should be done in the parent file.

Reply:
fair.